packages/https-proxy/test/helpers/certs/client/my-root-ca.crt.pem

```diff
@@ -1,24 +1,24 @@
------BEGIN CERTIFICATE-----

```

Comment:
Change from running `git add --renormalize . ` after adding .gitattributes file to normalize line endings